### PR TITLE
Local, Remote, and Relay Nodes.

### PR DESCRIPTION
**Pull request by andorsk**
Related to https://github.com/decentralized-identity/decentralized-web-node/issues/152

### CODE REVIEW

File: docs/companion_guide.md [hunk 1]
```diff
@@ -293,6 +294,31 @@ network performance and reliability. This section could also cover strategies
...
+  reachable from other services. If Bob takes a trip to the mountains and Jane
+  sends Bob a message, Jane would sent the message to Bob's remote DWN, which
+  is always available, rather than directly to Bob's phone, which is out of
+  connectivity. This allows Bob to still interact with Jane and receive her
+  message, despite not being connected to the internet.
+- **Relay:** The relay is a way a remote node to forward information received to a
+  local node, or another remote node. When Jane sends a message to Bob's remote
+  DWN, Bob's remote DWN "relay's" the message to Bob's local DWN, which allows
+  Bob to interact with his DWN locally.
+
```
Comment (TallTed) [inline docs/companion_guide.md:321]:
```suggestion
- **Relay:** A relay is a way for a remote node to forward information it receives to a
  local node, or to another remote node. When Jane sends a message to Bob's remote
  DWN, Bob's remote DWN "relays" the message to Bob's local DWN, which allows
  Bob to interact with his DWN locally.

```

**Review by TallTed — CHANGES_REQUESTED** [review]
Tweaks for clarity

**Comment by andorsk** [comment]
Let's discuss over the DIF call today and see if this can be merged.

**Comment by andorsk** [comment]
Thank you @TallTed for the tweaks! I've committed them. They look great.

**Review by LiranCohen — APPROVED** [review]

**Pull request merged**